src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3172,9 +3340,11 @@ class CgroupUtils:
                 pbs.logmsg(pbs.EVENT_DEBUG2,
                            "Offline file: %s" % self.offline_file)
                 try:
-                    with open(self.offline_file, 'w') as fd:
-                        fd.write('Offlined %s\n' % time.strftime("%c"))
+                    with open(self.offline_file, 'w') as desc:
+                        desc.write('Offlined %s\n' % time.strftime("%c"))
                 except:
+                    pbs.logmsg(pbs.EVENT_DEBUG2,
```

Comment:
We don't care about the exception type, only that there was a failure writing the file.

Reply:
This will catch KeyboardInterrupt.  This is what is sent to a hook that hit its alarm.  This creates a small race that if the alarm happens during this try/except block, you'll log a message and continue going.